### PR TITLE
Signin/display error message if email doesn't exist or bad password provided.

### PR DESCRIPTION
**Pull request by UserOlena**
- Sign In page can now receive an error from the server and display an appropriate error if email wasn't found in the DB or password didn't match the initial password stored in the DB.
- On Focus of either of the fields error message disappears (we can change it to a submit button event)

After Sign Up and Sign In PRs merged I will uncomment the line of code that redirects user to the home page on successful signup/signin.

### CODE REVIEW

**Comment by sendusyourbones** [comment]
I'm not seeing the error messages on the webpage. I think the syntax you used for your `switch case` (specifically the comma to include two cases at once) is not valid. From what I found online you can use:

```
case 'No email found!':
case 'Incorrect Password!':
    [code to execute for the above two cases]
```

Learnify

File: server/.envExample
Comment (sendusyourbones) [inline server/.envExample:1]: Same comment on this file as your other two PRs this one is built off of

**Comment by UserOlena** [comment]
> 

I think you are correct! I have no idea how it worked for me yesterday in both scenarios where email didn't exist or password was incorrect, but after restarting the server it doesn't. I looked that syntax up online and somehow it did work for me. I just searched for that syntax again and realized it was a question on the stackoverflow rather than the answer. It was after 2 am, so probably my brain wasn't working well. Sorry about that.

File: server/.envExample
Comment (UserOlena) [reply server/.envExample:1]: Should be resolved after PR #112 will be merged

**Review by sendusyourbones — APPROVED** [review]
Works great!

**Pull request merged**